public/data/KendallDoesCoding.json

```
@@ -22,18 +28,19 @@
     {
       "name": "Quiz",
       "url": "https://kendalldoescoding.gq/quiz",
-      "icon": "globe"
+      "icon": "link"
     },
     {
       "name": "Look at my subdomains/projects",
       "url": "https://kendalldoescoding.gq/subdomains",
       "icon": "globe"
-    }
-  ],
+    },
+  ]
```

```suggestion
  ],
```
Add `,` after closing square bracket.